connect_four.py

```diff
@@ -37,11 +36,14 @@ def check_col(self, col: str) -> int:
         else:
```

Comment:
in the lines above, use can use you defined, also what is the meaning of col_i - unicode_A? could that just be set to 0 instead, since i don't think you're using the output in the case that the column is a valid column?

Reply:
I'll rename the col_i variable, but col_i - unicode_A gives me the index that the valid column has in the self.columns matrix. it's basically the ord(input column) - ord("A")